packages/manager/modules/pci/src/projects/project/instances/instance/instance.html

```diff
@@ -135,7 +135,10 @@
             <div class="mt-4">
                 <ovh-advices
                     service-type="instance"
-                    service-name="{{ ::$ctrl.instanceId }}"
+                    service-name="{{ ::$ctrl.projectId }}"
+                    query-params="{
```

Comment:
Strange/confusing because the service-type is `instance`, so service Name should be the couple `[projectId, instanceId]` no ?

Reply:
I added URL and urlParams to avoid confusion. For simple URLs, serviceName and serviceType can be used. For more complex URLs, URL and urlParams can be used. In this case URL will be "/cloud/:serviceName/instance/:instanceId". This avoids all the confusion.